### DISCUSSION TITLE
Offer an Auth-only plan

### DISCUSSION THREAD

**Discussion by borque** [ente-io/ente#1875] [Enhancements]
Hello!
I would like to support Ente auth, but currently I have no need for the photo-app.
What about offering an additional (optional) subscription for auth-only users for let‘s say 12[$/€] a year (something between Bitwardens 10/year and a max of 15)?
Thanks and greetings

**Comment by sv3nnie** [comment]
You can support ente through GitHub Sponsors: https://github.com/sponsors/ente-io

**Reply by borque** [reply]
Thanks I was not aware of this possibility.
But why is it necessary to provide my full address to Github to send the money? 
When buying software elsewhere it is usually enough to provide a name, a mailadress somtimes the postal code and country and then the money can be send by PayPal or credit card.
